lib/foods.js

```diff
@@ -4,37 +4,92 @@
 const herokuUrl = require("./heroku-url")
 const $ = require("jQuery")
 
+$('#newFoodForm').hide();
+
 function getFoods() {
   //counter = 1;
   $.ajax({
     type: "GET",
     url: herokuUrl() + "api/v1/foods",
     success: function(posts) {
-      posts.forEach(function(foodType){
+      posts.reverse().forEach(function(foodType){
         event.preventDefault;
         let button = `<td><button type='button' class='deleteButton' id='${foodType.id}'>Delete</button></td>`
           let toInsert = `<tr id=${foodType.id}><td>${foodType.name}</td><td>${foodType.calories}</td>${button}</tr>`;
-          $(".foodsTable").append(toInsert);
+        $(".foodsTable").append(toInsert);
       });
       deleteFoodListener();
+      createFoodButtonListener();
+      createFoodFormListener();
     }
   })
 }
 
 
 function deleteFoodListener() {
-$('.deleteButton').on('click', function() {
-  console.log('delete button pressed');
-  //debugger;
-
-$.ajax({
-  type: 'DELETE',
-  url: herokuUrl() + `api/v1/foods/${event.target.id}`,
-  success: $(`#${event.target.id}`).hide()
+  $('.deleteButton').on('click', function() {
+
+    $.ajax({
+      type: 'DELETE',
+      url: herokuUrl() + `api/v1/foods/${event.target.id}`,
```

Comment:
@bretfunk if we are using ES6 string interpolation we should use it for the full url. Shouldn't this work?
```javascript
    url: `herokuUrl()api/v1/foods/${event.target.id}`
```